rules/windows/defense_evasion_windows_firewall_profile_disabled.toml

```
@@ -0,0 +1,60 @@
+[metadata]
+creation_date = "2021/10/15"
+maturity = "production"
+updated_date = "2021/11/22"
+
+
+[rule]
+author = ["Austin Songer"]
+description = """
+Identifies when a user disables Windows Firewall Profile(s) to help evade defense.
```

```suggestion
Identifies when the Windows Firewall is disabled using PowerShell cmdlets, which attackers do to evade network
constraints, like internet and network lateral communication restrictions.
```
https://github.com/elastic/detection-rules/pull/1565#discussion_r730962876